x/tokenfactory/types/params.go

```diff
@@ -13,7 +13,8 @@ var (
 	KeyDenomCreationGasConsume = []byte("DenomCreationGasConsume")
 
 	// For choice, see: https://github.com/osmosis-labs/osmosis/pull/4983
-	DefaultCreationGasFee = 40_000_000
+	// Set to 1_000_000 so that the gas limit doesnot go over 25M limit that is set in our config file.
```

Comment:
of course, how does this sound? 
"Set to 1_000_000 so that the gas limit doesnot go over 25M limit that is set in our config file. 25M limit is only for local mempool purposes, and thus is only ran on check tx. 

Setting this to 1M is fine because 1M = 0.005osmo minimum gas fee any arbitrage tx should have and also`denomCreationGasFee` can be reset to any other value in the future as long as it stays within the mempool limit."

Reply:
i'm honestly fine with a comment saying
`// chosen as an arbitrary large number, less than the max_gas_wanted_per_tx` which is the true reason lol.